enclave/core/shm.h

```diff
@@ -6,23 +6,23 @@
 
 #include <openenclave/bits/types.h>
 
-typedef struct _shared_memory_pool
+typedef struct _shared_memory_arena
 {
     /* Buffer holding the shared memory pool */
     uint8_t* buffer;
     size_t capacity;
     size_t used;
-    struct _shared_memory_pool* next;
-} Shared_memory_pool;
+    struct _shared_memory_arena* next;
+} Shared_memory_arena;
```

Comment:
seems like this should be shared_memory_arena_t

Reply:
Will do